exercism/ocr-numbers/ocr-numbers.js

```diff
@@ -0,0 +1,207 @@
+var numberToString = {
+    0:
+        " _ \n" +
+        "| |\n" +
+        "|_|\n" +
+        "   ",
+    1:
+        "   \n" +
+        "  |\n" +
+        "  |\n" +
+        "   ",
+    2:
+        " _ \n" +
+        " _|\n" +
+        "|_ \n" +
+        "   ",
+    3:
+        " _ \n" +
+        " _|\n" +
+        " _|\n" +
+        "   ",
+    4:
+        "   \n" +
+        "|_|\n" +
+        "  |\n" +
+        "   ",
+    5:
+        " _ \n" +
+        "|_ \n" +
+        " _|\n" +
+        "   ",
+    6:
+        " _ \n" +
+        "|_ \n" +
+        "|_|\n" +
+        "   ",
+    7:
+        " _ \n" +
+        "  |\n" +
+        "  |\n" +
+        "   ",
+    8:
+        " _ \n" +
+        "|_|\n" +
+        "|_|\n" +
+        "   ",
+    9:
+        " _ \n" +
+        "|_|\n" +
+        " _|\n" +
+        "   "
+};
+
+//create reverse object
+var stringToNumber = {}
+
+for (var number in numberToString) {
+  stringToNumber[numberToString[number]] = number
+}
+
+
+
+var ocr = {
+    split: function(string) {
+        var NUMBER_HEIGHT = 4;
+        var rows = string.split("\n");
+        var concatenated = [];
+        for (var i = 0; i < rows.length; i += NUMBER_HEIGHT) { //index = index+
+            var concatenatedRow = [];
+            concatenatedRow = rows.slice(i, i + NUMBER_HEIGHT);
+            concatenated.push(concatenatedRow);
+        }
+        return concatenated.map(function(con) {
+            return this.getStrings(con);
+        }, this);
+    },
+    convertHelper: function(stuff) {
+        return 7;
```

Comment:
Why do you return 7 every time? Also, were you intending to use the parameter 'stuff' somewhere? At the moment, this function could be replaced with a variable with the value 7, or simply wherever it is being called, replaced with the literal '7'